addons/website_crm_partner_assign_commissions/models/res_partner.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResPartnerCommission(models.Model):
+    _name = 'res.partner.commission'
+    _description = 'Reseller commissions'
+
+    grade_id = fields.Many2one('res.partner.grade', 'Grade', required=True)
+    percentage = fields.Float('Commission Percentage', required=True)
```

Comment:
To be more consistent with the nomenclature of your members, you may change this to `rate` too